src/components/Fingerprint.tsx

```diff
@@ -0,0 +1,63 @@
+import { Color } from "../types";
+
+interface Props {
+  width: number;
+  height: number;
+  color: Color;
+  top: number;
+  left: number;
+  hide?: boolean;
+}
+
+export function Fingerprint({ width, height, color, top, left, hide }: Props) {
+  return (
+    <svg
```

Comment:
Is there any way to replace the svg with an img instead? I think svgs just have bad performance when being dragged :((

Reply:
What about image for the blob/fingerprint and then a layer with `background-blend-mode` with a specific colour